test/judgments/test33.xml

```diff
@@ -40,6 +40,8 @@
         <uk:year>2022</uk:year>
         <uk:number>1890</uk:number>
         <uk:cite>[2022] EWHC 1890 (Fam)</uk:cite>
+        <uk:caseNumber>RG22C50069</uk:caseNumber>
+        <uk:caseNumber>RG22C50057</uk:caseNumber>
```

Comment:
Schema does not currently permit `caseNumber`. I've [previously pondered](https://my.slack.com/archives/C02TP2L2Z0F/p1732204622573629?thread_ts=1732153171.501399&cid=C02TP2L2Z0F) whether caseNumber should be a type of thing

```
<uk:identifier type=caseNumber>RG22C50069</uk:identifier>
<uk:identifier type=caseNumber>RG22C50057</uk:identifier>
```

Reply:
But this is what's agreed in the ADR.